package.json

```diff
@@ -106,7 +106,7 @@
     "css-loader": "^3",
     "eslint": "^7.5.0",
     "eslint-config-prettier": "^6.11.0",
-    "eslint-import-resolver-typescript": "^2.2.0",
+    "eslint-import-resolver-typescript": "^1.1.1",
```

Comment:
This was only triggering warnings, but since v2.0 this plugin doesn't work well (at least without other changes I haven't dug into). Cf https://github.com/alexgorbatchev/eslint-import-resolver-typescript/issues/33.